### PR TITLE
add livez endpoint to pqs docs

### PR DESCRIPTION
**Pull request by dasormeter**
Document `/livez` status endpoint for PQS docs

### CODE REVIEW

File: docs/2.8.3/docs/query/pqs-user-guide.rst [hunk 1]
```diff
@@ -1056,6 +1056,20 @@ Operate PQS
...
+The health of the Scribe component that feeds data to the Postgres database can be monitored using the health check
+endpoint ``/livez``. The health check endpoint is available at port specified through ``--health-port`` when Scribe is launched:
+
+```
+    --health-port int                          HTTP port to use to expose application health info (default: 8080)
+```
+
+```
+    Expect: responseStatus("/livez") is Status.Ok
+```
```
[the remark this answers is not in the document]
Comment (marciw-da) [reply docs/2.8.3/docs/query/pqs-user-guide.rst:1071]: I reduced the whitespace between ``--health-port int`` and the "HTTP port" explanation. I don't think readers really expect the whitespace in the docs to be exactly the same, so I'm voting for readability over accuracy just this once 🙂 

But feel free to revert that!

File: docs/2.8.3/docs/query/pqs-user-guide.rst [hunk 2]
```diff
@@ -1056,6 +1056,20 @@ Operate PQS
 
 This section discusses common tasks when operating a PQS.
 
+Status Health Check
+======================================
```
[the remark this answers is not in the document]
Comment (marciw-da) [reply docs/2.8.3/docs/query/pqs-user-guide.rst:1060]: Or maybe "Health status check," but "Status health" doesn't quite seem right.

Also, use sentence case (capitalize the first word only) https://docs.google.com/document/d/153D6-gI0blsAvlrzReHVyz8RCHUQzzdKnxnpcADlt6c/edit#heading=h.74n7maw54ixb

[the remark this answers is not in the document]
Comment (marciw-da) [reply docs/2.8.3/docs/query/pqs-user-guide.rst:1060]: @carrielaben-da I guess this should really be "Check health" to make it parallel with the other headings in this section... but that seems less readable to me (almost like "check" is a noun). WDYT?

File: docs/2.9.0/docs/query/pqs-user-guide.rst [hunk 3]
```diff
@@ -1060,6 +1060,21 @@ Operate PQS
...
+The health of the Scribe component that feeds data to the Postgres database can be monitored using the health check
+endpoint ``/livez``. The health check endpoint is available at port specified through ``--health-port`` when Scribe is launched:
+
+```
+    --health-port int                          HTTP port to use to expose application health info (default: 8080)
+```
+
+```
+    Expect: responseStatus("/livez") is Status.Ok
+```
```
[the remark this answers is not in the document]
Comment (marciw-da) [reply docs/2.9.0/docs/query/pqs-user-guide.rst:1075]: See comment on 2.8.3 change

**Review by marciw-da — COMMENTED** [review]
Made some suggestions. The only critical one is changing the backticks for the code blocks (use ``..code-block::`` instead)

File: docs/2.8.3/docs/query/pqs-user-guide.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (dasormeter) [reply docs/2.8.3/docs/query/pqs-user-guide.rst]: renamed it to `Check Health`

**Comment by dasormeter** [comment]
fyi @benm-da this PR should address the `Client's` formatting issue

**Comment by garyverhaegen-da** [comment]
> fyi @benm-da this PR should address the `Client's` formatting issue

That's already addressed by #677, I believe.

**Pull request merged**